gulp-tasks/build.js

```diff
@@ -43,6 +45,28 @@ const buildPackage = (projectPath) => {
     });
 };
 
+gulp.task('build:shared', () => {
```

Comment:
Just to confirm: this `umd` bundle is being consumed by the test, and not anything else, right?